packages/react/src/components/Search/next/Search-test.js

```diff
@@ -13,6 +13,23 @@ import { mount, shallow } from 'enzyme';
 const prefix = 'cds';
 
 describe('Search', () => {
+  let wrapper;
+
+  // const container = () => wrapper.find(`.${prefix}--search`);
+  const button = () => wrapper.find('button');
+  const input = () => wrapper.find('input');
+  // const label = () => wrapper.find('label');
+
+  const render = (props) => {
+    if (wrapper) {
+      return wrapper.setProps(props);
+    }
+
+    wrapper = mount(<Search labelText="testlabel" {...props} />);
+
+    return wrapper;
+  };
+
```

Comment:
It could be a `beforeAll`, but that would only be doing the same thing as this but with less clarity: the functions would need to be declared as `let` variables and then assigned in the `before` routine.

This code solves for an issue that I see a lot with front-end tests and which I see with the tests in this suite: if you render the element under test in a `beforeEach` routine, then it makes any other setup in nested contexts very difficult. You end up with multiple renders of the element.

In fact, I believe that some of the tests in this suite are finding the wrong element rendered.

If you move the mounting or rendering into each test/`it`, then you avoid this issue. 

To avoid duplicating the possibly complex mount/render declaration, I have adopted this paradigm which works quite well in my experience.

Each nested context/describe can build on the props:
* default: props reset
  * with size large: props.large = true
  * with size small: props.small = true
* with onChange: props.onChange = jest.fn();
* etc.

Each test simply becomes
```js
render(props);
expect(...);
```